package-lock.json

```diff
@@ -1,12270 +1,46 @@
 {
   "name": "architecture-builder",
   "version": "0.2.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@camjc here again with the new npm engine inclusion

Reply:
🤷